src/components/implementation/capmgr/naive/init.c

```diff
@@ -72,6 +73,26 @@ capmgr_comp_info_iter(void)
 		} else if (cos_spd_id() == spdid) {
 			cap_info_initthd_init(rci, sl__globals()->sched_thd, 0);
 		}
+
+		/*
+		 * Set tls region for this component
+		 * Expand the 2nd level pte within this component's page table at our new range
+		 */
+		tmp.pgtbl_cap = pgtslot;
+		tmp.memsrc    = ci;
+		ret = (int)cos_pgtbl_intern_alloc(&tmp, pgtslot, TLS_BASE_ADDR,
+					     PAGE_SIZE * TLS_NUM_PAGES);
+		assert(ret);
+
+		/* Place desired number of pages at this new range */
+		offset = 0;
+		for (i = 0 ; i < (signed)TLS_NUM_PAGES ; i++) {
```

Comment:
`(signed)` isn't telling what the type of `i` or the cast is.. being more explicit, like `(int)`?